src/stories/packages/components/Switch.stories.ts

```diff
@@ -0,0 +1,48 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { Switch } from '@terra-ui-packages/components';
+
+const meta: Meta<typeof Switch> = {
+  title: 'Packages/Components/Switch',
+  component: Switch,
+  parameters: {
+    layout: 'centered',
+  },
+  tags: ['autodocs'],
+  argTypes: {
+    checked: {
+      control: 'boolean',
+      description: 'state of the switch',
+      table: {
+        defaultValue: { summary: 'false' },
+      },
+    },
+    disabled: {
+      control: 'boolean',
+      description: 'disable the switch',
+      table: {
+        defaultValue: { summary: 'false' },
+      },
+    },
+    onLabel: {
+      control: 'text',
+      description: 'text for the "on" state',
+    },
+    offLabel: {
+      control: 'text',
+      description: 'text for the "off" state',
```

Comment:
I suggest adding documentation of `onChange` similar to https://github.com/DataBiosphere/terra-ui/blob/9ecb41a1442af71cd753950ea25923ea3f7725d5/src/stories/packages/components/Modal.stories.tsx#L79